jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -401,6 +488,15 @@ class JWTTools(
      * entries in the DIDDocument
      *
      */
+    @Suppress("DEPRECATION")
+    @Deprecated(
+        "Verifying a jwt token using the Universal Resolver is deprecated " +
+                "in favor of using a local resolver passed in as a method parameter." +
+                "This will be removed in the next major release.",
+        ReplaceWith(
+            """resolveAuthenticator(alg: String, issuer: String, auth: Boolean, resolver: DIDResolver)"""
+        )
+    )
     internal suspend fun resolveAuthenticator(alg: String, issuer: String, auth: Boolean): List<PublicKeyEntry> {
```

Comment:
This is an `internal` method so it does not need to be deprecated. It can directly be replaced with the new method signature that requires a `resolver`.